Source/Fuse.Nodes/Visual.BeginRemove.uno

```diff
@@ -84,7 +84,7 @@ namespace Fuse
 			}
 			else
 			{
-				args.Remove();				
+				UpdateManager.AddDeferredAction(args.Remove);
```

Comment:
Yes, the tests have to call PumpDeferred. Virtuallly all code had to work with removal being async anyways, due to RemovingAnimation. So it's not as scary as it seems.  Most places probably already assumed it was async.